client/state/selectors/is-google-my-business-stats-nudge-visible.js

```diff
@@ -21,11 +25,18 @@ export default function isGoogleMyBusinessStatsNudgeVisible( state, siteId ) {
 	// We don't want to show the nudge, and then hide it when it's obvious
 	// the site is actually already connected, therefore we must wait for the site
 	// settings to be fetched so we could verify site does not have connection connected
-	if ( getSiteSettings( state, siteId ) === null || isRequestingSiteSettings( state, siteId ) ) {
+	if ( getSiteKeyrings( state, siteId ) === null || isRequestingSiteKeyrings( state, siteId ) ) {
```

Comment:
For a site without keyrings  (not connected) that will look like:
![screen shot 2018-05-22 at 19 09 41](https://user-images.githubusercontent.com/326402/40375379-15ac6d76-5df4-11e8-9270-bb405be876a0.png)

So it wouldn't work 😭

Reply:
https://github.com/Automattic/wp-calypso/pull/24872/commits/058c8fa970c80b4efba764fd75d5d7c27d3ee443 fixes that